components/helm-broker/internal/bundle/repository_config.go

```diff
@@ -0,0 +1,36 @@
+package bundle
+
+import (
+	"path"
+	"path/filepath"
+	"strings"
+)
+
+// RepositoryConfig provides configuration for HTTP Repository.
+type RepositoryConfig struct {
+	BaseURL string `json:"baseUrl" valid:"required"`
+}
+
+// GetIndexFile return name of yaml file with configuration (if not exist return default name)
+func (cfg RepositoryConfig) GetIndexFile() string {
```

Comment:
shouldn't be more like `GetIndexFileName`?  `GetIndexFile` for me is more like getting the file content itself. In Go we also not using the prefix for Getter/Setters. More idiomatic it can be just `IndexFileName` but I'm also good with your approach ;)